<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название модели</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name"/>
      </item>
     </layout>
    </item>
    <item row="11" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Дата последнего завоза</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="delivery"/>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="price"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Вместимость</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="capacity"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Максимальная скорость</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxSpeed"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Разгон до 100 км/ч</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="accelerationTo100"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Грузоподъёмность</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="loadCapacity"/>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Количество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="count"/>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QCheckBox" name="climateControl">
        <property name="text">
         <string> Наличие системы климат-контроля</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Объём двигателя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="engineCapacity"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="type">
      <property name="title">
       <string>Тип</string>
      </property>
      <widget class="QRadioButton" name="cargo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>грузовой</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">tGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="passenger">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>легковой</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">tGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="bus">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>автобус</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">tGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="trailer">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>прицеп</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">tGroup</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="Show1">
        <property name="text">
         <string>Показать запись 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Show2">
        <property name="text">
         <string>Показать запись 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Save1">
        <property name="text">
         <string>Сохранить запись 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Save2">
        <property name="text">
         <string>Сохранить запись 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
